src/periodic_wave.rs

```diff
@@ -170,8 +170,9 @@ impl PeriodicWave {
     // cf. https://webaudio.github.io/web-audio-api/#waveform-generation
     // note that sines are in the imaginary components
     //
-    // the current implementation is naive and could be improved using inverse FFT
-    // or table lookup on SINETABLE
+    // @note: Current implementation is naive and could be improved using inverse
+    // FFT or table lookup on SINETABLE. Such performance improvements should be
+    // however tested also against this implementation.
```

Comment:
True, but since this does not occur on the render thread it is really fine for now